graphstorm-processing/docs/source/usage/example.rst

```diff
@@ -0,0 +1,249 @@
+        --output-prefix /tmp/gsprocessing-example/ \
+        --num-output-files 4
+
+
+To finalize processing and to wrangle the data into the structure that
+DGL distributed partitioning expects, we need an additional step that
+guarantees the data conform to the expectations of DGL:
+
+.. code-block:: bash
+
+    gs-repartition --input-prefix /tmp/gsprocessing-example/
+
+
+Examining the job output
+------------------------
+
+Once the processing and re-partitioning jobs are done,
+we can examine the outputs they created.
+
+.. code-block:: bash
+
+    $ cd /tmp/gsprocessing-example
+    $ ls
+
+    edges/  launch_arguments.json  metadata.json  node_data/
+    node_id_mappings/  perf_counters.json  updated_row_counts_metadata.json
+
+We have a few JSON files and the data directories containing
+the graph structure, features, and labels. In more detail:
+
+* ``launch_arguments.json``: Contains the arguments that were used
+  to launch the processing job, allowing you to check the parameters after the
+  job finishes.
+* ``updated_row_counts_metadata.json``:
+  This file is meant to be used as the input configuration for the
+  distributed partitioning pipeline. ``repartition_files.py`` produces
+  this file using the original ``metadata.json`` file as input.
+* ``metadata.json``: Created by ``gs-processing`` and used as input
+  for ``repartition_files.py``, can be removed once that script has run.
+* ``perf_counters.json``: A JSON file that contains runtime measurements
+  for the various components of GSProcessing. Can be used to profile the
+  application and discover bottlenecks.
+
+The directories created contain:
+
+* ``edges``: Contains the edge structures, one sub-directory per edge
+  type.
+* ``node_data``: Contains the features for the nodes, one sub-directory
+  per node type.
+* ``node_id_mappings``: Contains mappings from the original node ids to the
+  ones created by the processing job. This mapping would allow you to trace
+  back predictions to the original nodes/edges.
+
+If the graph had included edge features they would appear
+in an ``edge_data`` directory.
+
+At this point you can use the DGL distributed partitioning pipeline
+to partition your data, as described in the
+`DGL documentation <https://docs.dgl.ai/guide/distributed-preprocessing.html#distributed-graph-partitioning-pipeline>`_
+
+To simplify the process of partitioning and training, without the need
+to manage your own infrastructure, we recommend using GraphStorm's
+`SageMaker wrappers <https://github.com/awslabs/graphstorm/wiki/scale-sagemaker>`_
+that do all the hard work for you and allow
+you to focus on model development.
+
+To run GSProcessing jobs on Amazon SageMaker we'll need to follow
+:doc:`/usage/distributed-processing-setup` to set up our environment
+and :doc:`/usage/amazon-sagemaker` to execute the job.
+
+
+.. rubric:: Footnotes
+
+
+.. [#f1] Note that this is just a hint to the Spark engine, and it's
```

Comment:
Maybe we can move the footnote in the `index.rst` here.

Reply:
That one refers to row counts per file, this one refers to the number of files (splits) being produced.